step-capstone/src/components/Event.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../styles/TimeLine.css';
+/*
+Takes data in this form:
+       {
+         finalized: true,
+         type: "event",
+         title: "Visiting Sherlock",
+         location: "221b Baker Street, London",
+         startDate: "8:30",
+         endDate: "10:30",
+         description: "Additional notes"
+       }
+*/
+export default function Flight( props ) {
+   return(
+       <div className="event double events" style={props.styleConfig}>
+           <p>{props.data.startDate.toLocaleString()} - {props.data.endDate.toLocaleString()}: {props.data.title}</p>
```

Comment:
At least to me, it would be more readable to have the Title first, then a seperate <p> for the date afterwards. Ditto on flight and hotel.

Reply:
haha didnt realize this would convert my html, a seperate paragraph for the date.

Reply:
Good point! I will change the order of the rendering!

Reply:
done